codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -140,6 +153,25 @@ private void setActualSubprotocol(String actualSubprotocol) {
         this.actualSubprotocol = actualSubprotocol;
     }
 
+    public long forceCloseTimeoutMillis() { return forceCloseTimeoutMillis; }
+
+    /**
+     * Flag to indicate if the closing handshake was initiated because of timeout.
+     * For testing only.
+     */
+    protected boolean isForceCloseComplete() { return forceCloseComplete; }
+
+    private void setForceCloseComplete() { forceCloseComplete = true; }
```

Comment:
either remove the method or fix the styling to use:

```java
private void setForceCloseComplete() {
    ....
}
```

I would personally just remove it.

Reply:
Removed.